### PR TITLE
Added create-github-secret workflow

### PR DESCRIPTION
**Pull request by stavbernazport**
# Description

Added a Port Action example which creates a GitHub Secret using a GitHub workflow.

## Added docs pages

Please also include the path for the added docs

- GitHub Workflow (`/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret`)

## Updated docs pages

Please also include the path for the updated docs

- Jenkins Workflow (`/create-self-service-experiences/setup-backend/jenkins-workflow/examples/scaffold-repositories-using-cookiecutter`)

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      "workflow": "<GITHUB_WORKFLOW_FILE>"
+    },
+    "trigger": "CREATE",
+    "description": "Creates a GitHub secret in my repository",
+    "requiredApproval": false
+  }
+]
+```
+
+:::note
```
Comment (hadar-co) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md:98]: Please move the admonition above the code snippet (when it's after the code, readers miss it when they execute)

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+              "secret_value": "${{ inputs.secret_value }}"
+            }
+          relations: "{}"
+          clientId: ${{ secrets.PORT_CLIENT_ID }}
+          clientSecret: ${{ secrets.PORT_CLIENT_SECRET }}
+          operation: UPSERT
+          runId: ${{ fromJson(inputs.port_payload).context.runId }}
+```
+
+6. Trigger the action from Port's UI.
```
Comment (hadar-co) [inline docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md:153]: Would be helpful to add a link here, like this:
Trigger the action from the [Self-service](https://app.getport.io/self-serve) tab of your Port application.

**Review by hadar-co — CHANGES_REQUESTED** [review]

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md (hunk 1 above)
Comment (stavbernazport) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md:98]: done :)

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md (hunk 2 above)
Comment (stavbernazport) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/create-github-secret.md:153]: done :)

**Review by hadar-co — APPROVED** [review]
Nice!

**Pull request merged**